Podcast/AppDelegate.swift

```diff
@@ -88,6 +99,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         return true
     }
     
+    @objc func beginInterruption() {
+        Player.sharedInstance.pause()
+        print("interrupted")
```

Comment:
Is this the print statement you are referring too? Maybe explain more in the meeting since I am a bit confused